pkg/restore/restore.go

```diff
@@ -1062,13 +1062,13 @@ func (ctx *context) restoreItem(obj *unstructured.Unstructured, groupResource sc
 					ctx.log.Infof("ServiceAccount %s successfully updated", kube.NamespaceAndName(obj))
 				}
 			default:
-				e := errors.Errorf("not restored: %s and is different from backed up version.", restoreErr)
+				e := errors.Errorf("error restoring %s: not possible since it already exists in the cluster but is different from the backed up version.", restoreErr)
```

Comment:
Keep in mind that `restoreErr` already says something like "<resource identifier> already exists in cluster". It would be fine to not even include `restoreErr` in `e` if we just want to restate it, but I would definitely ensure that the resource type, namespace, and name are included in the message here.